.env.test

```diff
@@ -12,5 +12,5 @@ EPOCH_TIMESTAMP=1568059200
 
 ROLLBAR_TOKEN=
 
-AURORA_BOOTSTRAP_ROLLBAR_TOKEN_FILE=rollerbar_token
-AURORA_BOOTSTRAP_DB_CRED_FILE=db_pass
+ROLLBAR_TOKEN_FILE=rollbar_token
+DB_CRED_FILE=db_pass
```

Comment:
what is the benefit of having this be in a file? would it be useful to store both the password and the username?

Reply:
FYI it comes from here https://github.com/getoutreach/cdc_snapshot_validation/blob/master/deployments/aurora-bootstrap.jsonnet#L125-L126

Reply:
They are store in vault: https://vault.outreach.cloud/ui/vault/secrets/deploy/show/aurora-bootstrap/app1d/login-properties